systemd/ocp-delete-mco-leases.service

```diff
@@ -0,0 +1,13 @@
+[Unit]
```

Comment:
we don't need that because we are doing it as part of bundle creation https://github.com/crc-org/snc/blob/release-4.18/createdisk.sh#L174-L185